src/main/java/lottery/view/OutputView.java

```diff
@@ -0,0 +1,41 @@
+package lottery.view;
+
+import lottery.domain.LotteryNumber;
+import lottery.domain.Lottery;
+import lottery.domain.WinningType;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OutputView {
+    private final PrintStream out = System.out;
+
+    public void showLotteries(List<Lottery> lotteries) {
+        out.format("%1$d개를 구매했습니다.", lotteries.size());
+        out.println();
+        lotteries.forEach(this::showLotteryNumbers);
+    }
+
+    private void showLotteryNumbers(Lottery lottery) {
+        out.print("[");
+        out.print(lottery.getNumbers().stream().map(LotteryNumber::toString).collect(Collectors.joining(", ")));
+        out.println("]");
```

Comment:
```suggestion
        String numbers = String.format(
            "[%s]",
            lottery.getNumbers()
                .stream()
                .map(LotteryNumber::toString)
                .collect(Collectors.joining(", "))
        );
        out.println(numbers);
```
이렇게 표현하면 더 깔끔했을 것 같아요~